examples/09_cricket_umpire/run.sh

```
@@ -0,0 +1,31 @@
+python cricket_umpire.py run --call-id cricket-umpire-agent &
+AGENT_PID=$!
+sleep 3
+
+echo ""
+echo "✅ Token server running (PID: $TOKEN_PID)"
+echo "✅ DRS agent running (PID: $AGENT_PID)"
+echo ""
+echo "Open http://localhost:5173 in your browser"
+echo ""
+
+trap "kill $TOKEN_PID $AGENT_PID 2>/dev/null" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)